### ISSUE TITLE
Mac App minimize at startup 

### ISSUE THREAD

**Issue by jeffreye** [AvaloniaUI/Avalonia#2061]
I package artifacts on Mac and create ControlCatalog Mac app [Dropbox Link here](https://www.dropbox.com/s/vq565nh8eu0pr2s/ControlCatalog.zip?dl=0)

- It minimizes immediately at startup. 
- closing the window does not quit the app.

**Comment by jeffreye** [comment]
Yes, it allow high resolution. [See second line from the last](https://www.dropbox.com/s/vq565nh8eu0pr2s/ControlCatalog.zip?dl=0&file_subpath=%2FControlCatalog.app%2FInfo.plist)

I tried your info.plist, and it still happens. Regard closing, I see the same thing.

**Comment by jeffreye** [comment]
I close the window, sometime it does not quit the app. When this happen, I have to force quit the app.

**Comment by danwalmsley** [comment]
> Yes, it allow high resolution. [See second line from the last](https://www.dropbox.com/s/vq565nh8eu0pr2s/ControlCatalog.zip?dl=0&file_subpath=%2FControlCatalog.app%2FInfo.plist)
> 
> I tried your info.plist, and it still happens. Regard closing, I see the same thing.

Thats strange when I ran it, it was rendering all pixelated

**Comment by danwalmsley** [comment]
> I close the window, sometime it does not quit the app. When this happen, I have to force quit the app.

Will investigate

**Comment by jeffreye** [comment]
In case you need this, I ran it on macos Sierra.
<img width="269" alt="image" src="https://user-images.githubusercontent.com/1737680/47819391-9777d780-dd17-11e8-8886-0e3fd87d7633.png">

**Comment by latop2604** [comment]
Same on Mojave for minimizing.
But Closing is ok
